src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Memory-Strings.json

```diff
@@ -2,85 +2,85 @@
   "stsjorbsmod:CharityMemory": {
     "NAME": "Charity",
     "DESCRIPTIONS": [
-      "#bPassive: Gain 1 #yStrength for every !G! gold you have. (Gain #g!S! Strength.)"
+      "#pPassive: Gain #b1 #yStrength for every #b!G! gold you have. (Gain #b!S! Strength.)"
     ]
   },
   "stsjorbsmod:ChastityMemory": {
     "NAME": "Chastity",
     "DESCRIPTIONS": [
-      "#bRemember: Gain !R! #yDexterity. Apply !W! #yWeak to ALL enemies. NL #bPassive: At the end of each turn, lose !D! #yDexterity and gain !B! #yBlock."
+      "#pRemember: Gain #b!R! #yDexterity. Apply #b!W! #yWeak to ALL enemies. NL #bPassive: At the end of each turn, lose #b!D! #yDexterity and gain #b!B! #yBlock."
     ]
   },
   "stsjorbsmod:DiligenceMemory": {
     "NAME": "Diligence",
     "DESCRIPTIONS": [
-      "#bRemember: Draw !D! cards. NL #bPassive: At the end of your turn, #yRetain !M!."
+      "#pRemember: Draw #b!D! cards. NL #bPassive: At the end of your turn, #yRetain #b!M!."
     ]
   },
   "stsjorbsmod:EnvyMemory": {
     "NAME": "Envy",
     "DESCRIPTIONS": [
-      "#bRemember: Gain !V! #yVulnerable. #bPassive: Whenever you target an enemy with a card, apply !M! #yVulnerable to that enemy."
+      "#pRemember: Gain #b!V! #yVulnerable. #bPassive: Whenever you target an enemy with a card, apply #b!M! #yVulnerable to that enemy."
     ]
   },
   "stsjorbsmod:GluttonyMemory": {
     "NAME": "Gluttony",
     "DESCRIPTIONS": [
-      "#bPassive: Whenever you kill a non-minion enemy, raise your Max HP by !M!. At the start of each turn, discard one card."
+      "#pPassive: Whenever you kill a non-minion enemy, raise your Max HP by #b!M!. At the start of each turn, discard #b1 card."
     ]
   },
   "stsjorbsmod:GreedMemory": {
     "NAME": "Greed",
     "DESCRIPTIONS": [
-      "#bPassive: Whenever you kill a non-minion enemy, gain !M! gold."
+      "#pPassive: Whenever you kill a non-minion enemy, gain #b!M! gold."
     ]
   },
   "stsjorbsmod:HumilityMemory": {
     "NAME": "Humility",
     "DESCRIPTIONS": [
-      "#bRemember: Gain !R! #yThorns. NL #bPassive: Double the amount of #yThorns you have. (Gain !P! Thorns.)"
+      "#pRemember: Gain #b!R! #yThorns. NL #bPassive: Gain an additional #b2 #yThorns. (Gain #b!P! Thorns.)"
```

Comment:
Don't need the parenthetical for the passive effect now that it's not a dynamic number (just `#bPassive: Gain an additional #b!P! #yThorns.` is fine)